install/terminal/select-dev-language.sh

```diff
@@ -40,6 +40,9 @@ if [[ -n "$languages" ]]; then
 		Java)
 			mise use --global java@latest
 			;;
+		Flutter)
+			source ~/.local/share/omakub/install/terminal/flutter.sh
+			;;
```

Comment:
I am presuming this will load and execute the script. Will try to figure out a way to test it locally. :thinking: